crm_lead_stage_monitoring/crm.py

```diff
@@ -35,9 +35,10 @@ def write(self, cr, uid, ids, vals, context=None):
             vals['last_activity_stage'] = fields.datetime.now()
         return super(CrmLead, self).write(cr, uid, ids, vals, context=context)
 
-    def copy(self, cr, uid, id, default=None, context=None):
+    def copy(self, cr, uid, rec_id, default=None, context=None):
```

Comment:
Ugh! The new API encourages the infamous use of a reserved word as a variable name :broken_heart:

Please use the decorator instead of disabling this style check. BTW what is MQT exactly?